wellcomecollection/wellcomecollection.org
# Do prev/next guide stops with shallow links

## What does this change?
Uses the `shallow` prop of a NextLink in conjunction with the client side `router` so that we don't have to `getServerSideProps` every time someone hits previous/next, because we can work it out with the existing data we got from the first time we visited the page.

This
1. Speeds the page up for the user
2. Uses less of the user's data
3. Has a useful side-benefit of storing the AudioPlayer's speed across stops, since it isn't being re-instantiated

## How to test
- Run locally with the [Prismic endpoint](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/services/prismic/fetch/index.ts#L32) set to `wellcomecollection-stage`
- Visit an [audio stop](http://localhost:3000/guides/exhibitions/ZrHvtxEAACYAWmfc/audio-without-descriptions/2) play the audio, change the speed
- Use the prev/next links and validate the player speed stays what it was on the new page
- Turn off JS and verify the page will still work

## How can we measure success?
Users visit more stops

## Have we considered potential risks?
Can't think of any
